cardano-cli/src/Cardano/CLI/Ops.hs

```diff
@@ -270,6 +271,7 @@ data CliError
   | UpdateProposalSubmissionError !RealPBFTError
   | VerificationKeyDeserialisationFailed !FilePath !Text
   | FileNotFoundError !FilePath
+  | CardanoApiError !ApiError
```

Comment:
@erikd: Should I define more specific errors? e.g. `ShelleyKeyPairGenerationError`.

Reply:
Probably only need one constructor for `ApiError`.